synfig-core/src/synfig/loadcanvas.cpp

```diff
@@ -2284,6 +2284,13 @@ CanvasParser::parse_static_list(xmlpp::Element *element,Canvas::Handle canvas)
 	return value_node;
 }
 
+bool read_bool_attribute(xmlpp::Element* element, const char* name) {
```

Comment:
- prepend `static`

- Maybe rename to `is_bool_attribute_true()` or `evaluate_bool_attribute()`
When I read `if (read_bool_attribute)` I have the impression I'm checking if it could read the attribute, not exactly evaluating its value.

Reply:
> I have the impression I'm checking if it could read the attribute

This is the correct impression, because it tries to read the attribute and returns `false` if it cannot read it (for example, the attribute was not found).

> Maybe rename to is_bool_attribute_true()

Agree. This is more correct.